src/Xamarin.Android.Build.Tasks/Tasks/GenerateLayoutBindings.CSharpBindingGenerator.cs

```diff
@@ -157,8 +157,35 @@ void WriteClassClose (State state)
 				WriteLineIndent (state, "}");
 			}
 
+			void WriteDisableWarnings (State state)
+			{
+				state.WriteLine ("#pragma warning disable CS0618");
```

Comment:
What members are we referencing which are obsolete ([CS0618](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs0618))?

Reply:
Good point, it was for PreserveAttribute but I removed them completely.

Reply:
Then should we remove this pragma?  Or do we want to slience obsolete warnings in this file anyway?